proposals/description/clawback.ts

```diff
@@ -0,0 +1,57 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const clawback: TemplatedProposalDescription = {
+  title: 'FIP-113: End Departed Rari Founders Vesting of TRIBE',
+  commands: [
+    /////////  Mint FEI and TRIBE onto new RARI timelocks
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: (addresses) => [addresses.newRariInfraFeiTimelock, '3178504756468797564687976'],
+      description: 'Mint FEI to the new Rari Infra FEI timelock'
+    },
+
+    {
+      target: 'core',
+      values: '0',
+      method: 'allocateTribe(address,uint256)',
+      arguments: (addresses) => [addresses.newRariInfraTribeTimelock, '3178502219685438863521056'],
+      description: 'Allocate TRIBE to the new Rari Infra TRIBE timelock'
+    },
+
+    ////  Clawbacks
+    {
+      target: 'clawbackVestingContractA',
+      values: '0',
+      method: 'clawback()',
+      arguments: (addresses) => [],
+      description: 'Clawback the TRIBE of Jai Bhavnani and David Lucid'
+    },
+    {
+      target: 'clawbackVestingContractB',
+      values: '0',
+      method: 'clawback()',
+      arguments: (addresses) => [],
+      description: 'Clawback the TRIBE of Jai Bhavnani and David Lucid'
+    },
+    {
+      target: 'clawbackVestingContractC',
+      values: '0',
+      method: 'clawback()',
+      arguments: (addresses) => [],
+      description: 'Clawback the TRIBE of Jai Bhavnani and David Lucid'
+    }
```

Comment:
the proposal should withdraw all the TRIBE from the timelock back to core after or at least approve some contract like the Tribal Council Timelock to do so

Reply:
Have approved the TC timelock for 20M and added checks that it was approved and can transfer